Makefile

```diff
@@ -143,6 +143,9 @@ ipackage-sim: clean Xcode/ios
 itest: ipackage-sim KIF
 	./scripts/ios/test.sh
 
+ianalyze: clean Xcode/ios
```

Comment:
Remove `clean` from the dependencies, this is adding several minutes to the build time.

I think we should remove it from `ipackage` and `ipackage-sim` too.